content/guides/docker-concepts/running-containers/sharing-local-files.md

```diff
@@ -0,0 +1,211 @@
+{ .tip }
+
+## Try it out
+
+In this hands-on guide, you’ll practice how to create and use a bind mount to share files between a host and a container.
+
+### Run a container
+
+1. [Download and install](https://www.docker.com/products/docker-desktop/) Docker Desktop
+
+2. Start a container using the [httpd](https://hub.docker.com/_/httpd) image with the following command:
+
+   ```console
+   $ docker run -d -p 8080:80 --name my_site httpd:2.4
+   ```
+
+   This will start the `httpd` service in the background, and publish the webpage to port `8080` on the host.
+
+3. Open the browser and access [http://localhost:8080](http://localhost:8080) or use the curl command to verify if it's working fine or not.
+
+    ```
+    $ curl localhost:8080
+    ```
+
+
+### Use Bind mount
+
+Using a bind mount, you can map the configuration file on your host computer to a specific location within the container. In this example, you’ll see how to change the look and feel of the webpage by using bind mount:
+
+1. Delete the existing container by using the Docker Dashboard:
+
+   ![A screenshot of Docker dashboard showing how to delete the httpd container](images/delete-httpd-container.webp?border=true)
+
+
+2. Create a new directory called `public_html` on your host system.
+
+    ```console
+    $ mkdir public_html
+    ```
+
+3. Change the directory to `public_html` and create a file `index.html` with the following content. This is a basic HTML document that creates a simple webpage that welcomes you with a friendly whale!
+
+    ```
+    <!DOCTYPE html>
+    <html lang="en">
+    <head>
+    <meta charset="UTF-8">
+    <title> My Website with a Whale & Docker!</title>
+    </head>
+    <body>
+    <h1>Whalecome!!</h1>
+    <p>Look! There's a friendly whale greeting you!</p>
+    <pre id="docker-art">
+       ##         .
+      ## ## ##        ==
+     ## ## ## ## ##    ===
+     /"""""""""""""""""\___/ ===
+   {                       /  ===-
+   \______ O           __/
+    \    \         __/
+     \____\_______/
+
+    Hello from Docker!
+    </pre>
+    </body>
+    </html>
+    ```
+
+4. It's time to run the container. The `--mount` and `-v` examples produce the same result. You can't run them both unless you remove the `my_site` container after running the first one.
+
+   {{< tabs >}}
+   {{< tab name="`-v`" >}}
+
+   ```console
+   $ docker run -d --name my_site \
+   -p 8080:80 \
+   -v .:/usr/local/apache2/htdocs/ \
+   httpd:2.4
+   ```
+
+   {{< /tab >}}
+   {{< tab name="`--mount`" >}}
+
+   ```console
+   $ docker run -d --name my_site \
+   -p 8080:80 \
+   --mount type=bind,source=./,target=/usr/local/apache2/htdocs/ \
+   httpd:2.4
+   ```
+
+   > **Tip**  
```

Comment:
I think the commands above won't work in PowerShell either because it uses a backtick (`) for multi-line commands.

One suggestion, if Docker recommends using the `--mount` option, then only show the `--mount` syntax and use the tabs to show the command for different shells.

Reply:
Hello @craig-osterhout I tested it on Windows. The only thing we would need is absolute path. I removed "\" from both of the commands and called out a tip that is common to both. Do check out the latest changes and let me know if that works. Also, we wanted to highlighted both `-v` and `--mount` flag for our readers.